qa/integration/services/monitoring_api.rb

```diff
@@ -21,47 +21,51 @@
 # Convenience class to interact with the HTTP monitoring APIs
 class MonitoringAPI
 
+  def initialize(port = 9600)
```

Comment:
same here... also, above says api_port and here port.

Reply:
_I am open to change it if requires._
My philosophy behind the naming is usually `subject.action` or `subject.property` (for both private and public members). For example, if we have a car class, it does have engine, wheels, etc.. So, any instance of car object members would be more readable with `car.engine`/`car.wheels` instead `car.car_engine`/`car.wheels`. For this case also, `MonitoringAPI` subject already defined API context, `monitoring_api.port` would be preferred for me instead `monitoring_api.api_port`.